### PR TITLE
workflows: fixes to make author dags work

### PR DESCRIPTION
**Pull request by DonHaul**

### CODE REVIEW

File: dags/author/author_create/author_create_approved.py [hunk 1]
```diff
@@ -89,18 +89,29 @@ def create_author_on_inspire(**context: dict) -> str:
         workflow_data = workflow_management_hook.get_workflow(
             workflow_id=context["params"]["workflow_id"]
         )
+
+        data = {
+            "$schema": "http://inspirebeta.net/schemas/records/authors.json",
+            "name": {"value": "PlaceHolder"},
```
Comment (drjova) [inline dags/author/author_create/author_create_approved.py]: why do we need a placeholder?

File: dags/author/author_create/author_create_approved.py [hunk 2]
```diff
@@ -89,18 +89,29 @@ def create_author_on_inspire(**context: dict) -> str:
         workflow_data = workflow_management_hook.get_workflow(
             workflow_id=context["params"]["workflow_id"]
         )
+
+        data = {
+            "$schema": "http://inspirebeta.net/schemas/records/authors.json",
```
Comment (drjova) [inline dags/author/author_create/author_create_approved.py]:
```suggestion
            "$schema": "http://inspirehep.net/schemas/records/authors.json",
```

File: dags/author/author_create/author_create_approved.py [hunk 3]
```diff
@@ -89,18 +89,29 @@ def create_author_on_inspire(**context: dict) -> str:
...
+            "name": {"value": "PlaceHolder"},
+            "_collections": ["Authors"],
+        }
+        # TODO Convert from submission to proper inspire beta schema
+        data["name"]["value"] = (
+            workflow_data["data"]["data"]["given_name"]
+            + " "
+            + workflow_data["data"]["data"]["family_name"]
+        )
```
Comment (drjova) [inline dags/author/author_create/author_create_approved.py]: I don't think we need this

**Review by drjova — CHANGES_REQUESTED** [review]
Thank you, I have added some comments

File: dags/author/author_create/author_create_approved.py (hunk 3 above)
Comment (DonHaul) [reply dags/author/author_create/author_create_approved.py]: Removed

File: dags/author/author_create/author_create_approved.py (hunk 1 above)
Comment (DonHaul) [reply dags/author/author_create/author_create_approved.py]: According to the name property must have a value, it must be a non empty string

File: dags/author/author_create/author_create_approved.py [hunk 4]
```diff
@@ -48,6 +47,7 @@ def author_create_approved_dag():
 
     @task()
     def set_workflow_status_to_running(**context):
+        """sets status to running"""
```
Comment (drjova) [inline dags/author/author_create/author_create_approved.py]: we don't need this docstring

File: dags/author/author_create/author_create_approved.py [hunk 5]
```diff
@@ -86,21 +86,26 @@ def create_author_create_curation_ticket(**context: dict) -> None:
...
     @task(do_xcom_push=True)
     def create_author_on_inspire(**context: dict) -> str:
-        workflow_data = workflow_management_hook.get_workflow(
+        workflow_management_hook.get_workflow(
             workflow_id=context["params"]["workflow_id"]
         )
+
+        data = {
+            "$schema": "http://inspirehep.net/schemas/records/authors.json",
+            "name": {"value": "NAME"},
```
Comment (drjova) [inline dags/author/author_create/author_create_approved.py]: Why do we need that?

**Review by drjova — CHANGES_REQUESTED** [review]
Thank you, two small comments

File: plugins/hooks/backoffice/workflow_ticket_management_hook.py [hunk 6]
```diff
@@ -35,7 +35,7 @@ def get_ticket(self, workflow_id: str, ticket_type: str) -> dict:
     def create_ticket_entry(
         self, workflow_id: str, ticket_id: str, ticket_type: str
     ) -> Response:
-        endpoint = f"/workflow-ticket/"
+        endpoint = "api/workflow-ticket/"
```
Comment (drjova) [inline plugins/hooks/backoffice/workflow_ticket_management_hook.py:38]: are you sure we don't need the `/` in the beginning?

Comment (DonHaul) [reply plugins/hooks/backoffice/workflow_ticket_management_hook.py:38]: Currently the hook handles it for use if there is no `/`
https://github.com/inspirehep/workflows/blob/main/plugins/hooks/backoffice/base.py#L49
If it makes it more clear I can explicitly add it to every endpoint

**Review by drjova — APPROVED** [review]

**Comment by drjova** [comment]
@DonHaul could you please squash?

**Comment by DonHaul** [comment]
@drjova could you give me write access? I cant seem to merge

**Comment by drjova** [comment]
@DonHaul could you please rebase?

**Review by drjova — APPROVED** [review]

**Pull request merged**